Dev-Tools/basecamp-connector/tool.md

```diff
@@ -0,0 +1,98 @@
+# Google Forms ➡️ Basecamp
+
+## Overview
+
+This project started out as a simple idea that we could use the "onsubmit" action on a Google form to trigger an update in Basecamp that would create a card detailing what needed to be done. Originally Westin had the idea for a few forms that the team was manually transferring information from Forms to Basecamp for, but the project became more complex than that in the end.
+
+We now have the capability to create both todo list items and cards on the card table, and will soon have a feature to allow the connector to assign people to a card by default.
+
+## Creating Connectors
+
+### Setting up the environment
+
+Thankfully the Google Apps Script environment has everything that you would need for making a device like this.
+There aren't too many things you have to do to set up the new project. The most important thing is making sure that you have edit access to the form.
+Without this edit access, you will not be able to create a script that will run on form submissions.
+
+The next thing to do is to link the library that we created to facilitate easier creation of these connectors.
+
+![Add library button](./images/add_cloudflare_library.png)
+
+Once the dialog opens and you see the text box, enter `1klv2AvLhl7gGeBTJjItWbnLeKS5gRcg1jWXEpmchs27gKkBxV1gvlFaG` and press add.
```

Comment:
Should this be private?

Reply:
This does not need to be private as the library is only available to people we share it with.